core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -61,6 +61,26 @@ <h3>Regenerate contribution opportunities for a topic:</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Get Exploration Interactions</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.retrieveExplorationInteractionIds()">
+      <div class="form-group row">
+        <label>Exploration ID:</label> <input type="text" ng-model="$ctrl.explorationId" required="true">
+      </div>
+      <div class="form-group row">
+        <input type="submit" value="submit query">
+      </div>
+    </form>
+    <p ng-if="$ctrl.explorationInteractionIds">
```

Comment:
Will it pass if `explorationInteractionIds` is an empty list?

Reply:
i don't think so

Reply:
Can you please check?

Reply:
Yes it's is working for empty list as well

Reply:
Should we maybe change the UI based on if it's empty? It might be worthwhile in the event of an empty list to show something like: "Interaction IDs: None"